### PR TITLE
RHDEVDOCS-3868 - Review RNs for OCP 4.10.z bug fix update

### PR DESCRIPTION
**Pull request by rishumehra**
* Aligned team: Dev Tools
* For branches: Enterprise 4.10.z release notes
* Jira: [RHDEVDOCS-3868](https://issues.redhat.com/browse/RHDEVDOCS-3868)
* Direct link to doc preview: https://deploy-preview-44809--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-10-release-notes#ocp-4-10-bug-fixes
* SME review: Received in the individual BZs.
* QE review: @sanketpathak
* Peer review: @Srivaralakshmi @max-cx

### CODE REVIEW

File: release_notes/ocp-4-10-release-notes.adoc [hunk 1]
```diff
@@ -2068,6 +2068,8 @@ In various circumstances, the management state of the Cluster Samples Operator c
 
 * Before this update, the tooltip for tasks in the *PipelineRun details* showed misleading information. It showed the time elapsed since the task ran, not how long they ran. For example, it showed 122 hours for a task that ran for a couple of seconds 5 days ago. With this update, the tooltip shows the duration of the task. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2011368[BZ#2011368])
 
+* Before this update, when you use the *Import from Git* form to import a private git repository, the correct import type and the builder image are not identified. This issue used to happen because the secret to fetch the private repository details is not decoded. This update resolves the issue by decoding the secret value before using it. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2057507[BZ#2057507])
```
Comment (Srivaralakshmi) [inline release_notes/ocp-4-10-release-notes.adoc:2071]:
```suggestion
* Before this update, when a secret is provided for a private repository in the *Import from Git* form, the value of the secret to fetch the private repository details is not decoded before being used. As a result, when you use the *Import from Git* form to import a private Git repository, the correct import type, and the builder image are not identified. This update resolves the issue by decoding the secret value before using it. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2057507[BZ#2057507])
```

**Review by Srivaralakshmi — CHANGES_REQUESTED** [review]
@rishumehra PTAL. Thanks!

File: release_notes/ocp-4-10-release-notes.adoc (hunk 1 above)
Comment (rishumehra) [reply release_notes/ocp-4-10-release-notes.adoc]: Accepting the changes. Thanks @Srivaralakshmi

**Review by max-cx — APPROVED** [review]
LGTM after you review my suggested change in https://github.com/openshift/openshift-docs/pull/44809/files#r857438220

**Review by max-cx — APPROVED** [review]

File: release_notes/ocp-4-10-release-notes.adoc [hunk 2]
```diff
@@ -2068,6 +2068,7 @@ In various circumstances, the management state of the Cluster Samples Operator c
 
 * Before this update, the tooltip for tasks in the *PipelineRun details* showed misleading information. It showed the time elapsed since the task ran, not how long they ran. For example, it showed 122 hours for a task that ran for a couple of seconds 5 days ago. With this update, the tooltip shows the duration of the task. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2011368[BZ#2011368])
 
+* Before this update, when a secret was provided for a private repository in the *Import from Git* form, the value of the secret to fetch the private repository details was not decoded. As a result, when using the *Import from Git* form to import a private Git repository, the import type and builder image were not detected. This update resolves the issue by decoding the secret value before using it. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2057507[BZ#2057507])
```
Comment (opayne1) [inline release_notes/ocp-4-10-release-notes.adoc:2071]: Not sure about the use of *Import from Git* form. I am not so sure it shouldn't just be *Git import* I'll have someone verify. I made a few suggestions based on minimalism and clarity.
```suggestion
* Before this update, import strategy detection did not occur when a secret was provided for a private Git repository. Consequently, the secrets value was not decoded before it is used. With his update, the secret value is now decoded before its use. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2057507[BZ#2057507])
```

**Review by opayne1 — COMMENTED** [review]
Just a few comments before we can merge! :)

**Comment by opayne1** [comment]
@rishumehra You will need to fetch/rebase your `enterprise-4.10` branch and add that bug doc text under the bug fix section in 4.10.8 z-stream entry. See https://docs.openshift.com/container-platform/4.10/release_notes/ocp-4-10-release-notes.html#ocp-4-10-8

**Comment by opayne1** [comment]
LGTM!

File: release_notes/ocp-4-10-release-notes.adoc [hunk 3]
```diff
@@ -2753,6 +2753,8 @@ Starting with {product-title} 4.10.8, support for Google Cloud Platform Workload
 
 * Previously, the Local Storage Operator (LSO) added an `OwnerReference` object to the persistent volumes (PV) it created, which sometimes caused an issue where a delete request for a PV could leave the PV in the `terminating` state while still attached to the pod. With this update, the LSO no longer creates an `OwnerReference` object and cluster administrators are now able to manually delete any unused PVs after a node is removed from the cluster. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2065714[*BZ#2065714*])
 
+* Before this update, import strategy detection did not occur when a secret was provided for a private Git repository. Consequently, the secrets value was not decoded before it was used. With his update, the secret value is now decoded before use. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2057507[*BZ#2057507*])
```
Comment (sanketpathak) [inline release_notes/ocp-4-10-release-notes.adoc:2756]:
```suggestion
* Before this update, import strategy detection did not occur when a secret was provided for a private Git repository. Consequently, the secrets value was not decoded before it was used. With this update, the secret value is now decoded before use. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2057507[*BZ#2057507*])
```

Comment (rishumehra) [reply release_notes/ocp-4-10-release-notes.adoc]: Done

File: release_notes/ocp-4-10-release-notes.adoc [hunk 4]
```diff
@@ -2068,6 +2068,8 @@ In various circumstances, the management state of the Cluster Samples Operator c
 
 * Before this update, the tooltip for tasks in the *PipelineRun details* showed misleading information. It showed the time elapsed since the task ran, not how long they ran. For example, it showed 122 hours for a task that ran for a couple of seconds 5 days ago. With this update, the tooltip shows the duration of the task. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2011368[BZ#2011368])
 
+* Before this update, when a secret is provided for a private repository in the *Import from Git* form, the value of the secret to fetch the private repository details is not decoded before being used. As a result, when you use the *Import from Git* form to import a private Git repository, the correct import type, and the builder image are not identified. This update resolves the issue by decoding the secret value before using it. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2057507[BZ#2057507])
```
[the remark this answers is not in the document]
Comment (rishumehra) [reply release_notes/ocp-4-10-release-notes.adoc]: Done

**Review by RickJWagner — APPROVED** [review]
Looks good.

File: release_notes/ocp-4-10-release-notes.adoc (hunk 2 above)
Comment (rishumehra) [reply release_notes/ocp-4-10-release-notes.adoc]: done

**Review by bergerhoffer — COMMENTED** [review]
LGTM too, merging per @opayne1's ping.

**Pull request merged**